assets/src/app/order/orders.component.ts

```diff
@@ -151,14 +146,14 @@ export class OrdersComponent {
 
   updateWorkflow(workflow_id) {
     this.workflowService.getWorkflow(workflow_id)
-    .subscribe(workflowData => {
-      for (let i = 0; i < this.workflows.data.length; i++) {
-        if (this.workflows.data[i].id === workflowData.data.id) {
-          this.workflows.data[i] = workflowData.data
-          return
+      .subscribe(workflowData => {
```

Comment:
In the method `eventGetWorkflows`, the `this.getWorkflows()` has no parameters but now the service requires 3 parameters ==> compiler error AGAIN :'(